ui/src/VoxettaApp.js

```diff
@@ -30,3 +26,36 @@ export class VoxettaApp extends LitElement {
 }
 
 customElements.define('vox-app', VoxettaApp);
+
+
+navigator.mediaDevices.getUserMedia({ audio: true, video: false })
+    .then(function(stream) {
+
+        const recordButton = document.getElementById("recordButton");
+        const audioSave = document.getElementById("utterance");
+        let isRecording = false;
+        let recordedChunks = [];
+        const mediaRecorder = new MediaRecorder(stream);
+        
+        recordButton.addEventListener('click', (ev)=>{
```

Comment:
Check the formatting here

https://google.github.io/styleguide/jsguide.html#formatting

Incorrect formatting on arrow function, if/else statement